HOMEWORK/AliaksandrZakharevich/hw_6.js

```diff
@@ -0,0 +1,165 @@
+                  <label class="col-form-label" id="labelAll">All:0</label>
+                </div>
+                <div class="col-auto">
+                  <label class="col-form-label" id="labelCompleted">Completed:0</label>
+                </div>
+                <div class="col">
+                  <button class="btn btn-primary AZ-w-100" id="showAll">Show All</button>
+                </div>
+                <div class="col">
+                  <button class="btn btn-primary AZ-w-100" id="showComplete">Show Completed</button>
+                </div>
+                <div class="col">
+                  <input type="text" id="inputSearch" class="p-2 text-center AZ-input" placeholder="Search...">
+                </div>
+              </div>
+            </div>
+        `
+    }
+
+     const pushTodoList = () =>{
+        const main = document.querySelector('.main');
+        if(todoArray.length > 0) {
+            for (let i = todoArray.length-1; i < todoArray.length; i++) {
+            const div = createNode({nodeName: 'div', nodeClassName: ['row', 'p-2']})
+                div.innerHTML = `
+                  <div class="container col-8 p-2 mb-3 AZ-list-container todo${i}">
+                    <div class="row align-items-center">
+                      <div class="col-auto">
+                        <button type="button" class="btn btn-primary AZ-btn-p AZ-btn-m-left" id="btnCompleteTodo${i}" data-bs-toggle="button">&#10003;</button>
+                      </div>
+                      <div class="col">
+                        <label class="d-block p-3 text-center AZ-input" id="labelTodo${i}">Search...</label>
+                      </div>
+                      <div class="col-auto">
+                        <button type="button" class="btn btn-primary d-block AZ-btn-p AZ-btn-m-rigth" id="btnDeleteTodo${i}">&#10005;</button>
+                        <label class="col-form-label d-block AZ-date-container" id="dateTodo${i}">data</label>
+                      </div>
+                    </div>
+                  </div>
+            `;
+                main.append(div);
+                const label = document.querySelector(`#labelTodo${i}`);
+                const data = document.querySelector(`#dateTodo${i}`);
+                const checkBtn = document.querySelector(`#btnCompleteTodo${i}`);
+                label.textContent = todoArray[i].todoTheme;
+                data.textContent = todoArray[i].data;
+                checkBtn.onclick = completeTodo;
+                // const btnDeleteTodo = document.querySelector(`.btnDeleteTodo${i}`);
+                // btnDeleteTodo.onclick = deleteTodo;
+            }
+            countTodo();
+        }
+    }
+
+    const addTodo = () =>{
+        let todoName = document.querySelector('#inputEnter');
+        if(todoName.value){
+            const data = new Date().toLocaleString();
+            todoArray.push({todoTheme: `${todoName.value}`, data: data, isCompleted: false})
```

Comment:
In english better to name it as "topic" not a  "theme". Theme is more about appearance